huxley/www/templates/www.html

```diff
@@ -19,7 +19,7 @@
             ContactTypes = {{ contact_types|safe }};
             ProgramTypes = {{ program_types|safe }};
         </script>
-        {% javascript 'huxley' %}
+        <script type="text/javascript" src="/static/js/bundle.js"></script>
```

Comment:
It's not an absolute path; in HTML it's a relative URL. So, if the domain were stage.huxley.bmun.org, it would resolve to stage.huxley.bmun.org/static/js/bundle.js. The /static/ path maps to the static applications on the webserver, which will serve the static file.